kratos/processes/simple_mortar_mapper_process.h

```diff
@@ -0,0 +1,474 @@
+           mOriginVariable(OriginVariable),
+           mDestinyVariable(DestinyVariable)
+    {
+    }
+
+    /// Destructor.
+    ~SimpleMortarMapperProcess() override
+    = default;
+
+    ///@}
+    ///@name Access
+    ///@{
+
+    ///@}
+    ///@name Inquiry
+    ///@{
+
+    ///@}
+    ///@name Input and output
+    ///@{
+
+    ///@}
+    ///@name Friends
+    ///@{
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+    void operator()()
+    {
+        Execute();
+    }
+    
+    ///@}
+    ///@name Operations
+    ///@{
+    
+    void Execute() override
+    {
+        KRATOS_TRY;
+
+        // Defining tolerance
+        const double tolerance = std::numeric_limits<double>::epsilon();
+        
+        // We compute the nodal area
+        ComputeNodalArea();
+        
+        // Create and initialize condition variables:
+        MortarKinematicVariables<TNumNodes> rVariables;
+    
+        // Create the mortar operators
+        MortarOperator<TNumNodes> rThisMortarConditionMatrices;
+        
+        // We call the exact integration utility
+        ExactMortarIntegrationUtility<TDim, TNumNodes> integration_utility = ExactMortarIntegrationUtility<TDim, TNumNodes>(TDim);
+        
+        // Initialize to zero // TODO: Add OMP
+        for(int i=0; i<static_cast<int>(mrThisModelPart.Conditions().size()); i++)
+        {
+            auto it_cond = mrThisModelPart.ConditionsBegin() + i;
+            
+            if (it_cond->Is(SLAVE) == true)
+            {
+                boost::shared_ptr<ConditionMap>& all_conditions_maps = it_cond->GetValue( MAPPING_PAIRS );
+                
+                for (auto it_pair = all_conditions_maps->begin(); it_pair != all_conditions_maps->end(); ++it_pair )
+                {
+                    Condition::Pointer p_cond_master = (it_pair->first); // MASTER
+                    GeometryType& master_geometry = p_cond_master->GetGeometry();
+                    
+                    for (unsigned int i_node = 0; i_node < TNumNodes; i_node++)
+                    {
+                        MortarUtilities::ResetValue<TVarType, THist>(master_geometry[i_node], mDestinyVariable);
+                    }
+                }
+            }
+        }
+        
+        // We map the values from one side to the other // TODO: Add OMP
+        for(int i=0; i<static_cast<int>(mrThisModelPart.Conditions().size()); i++)
+        {
+            auto it_cond = mrThisModelPart.ConditionsBegin() + i;
+            
+            if (it_cond->Is(SLAVE) == true)
+            {
+                const array_1d<double, 3>& slave_normal = it_cond->GetValue(NORMAL);
+                GeometryType& slave_geometry = it_cond->GetGeometry();
+                
+                boost::shared_ptr<ConditionMap>& all_conditions_maps = it_cond->GetValue( MAPPING_PAIRS );
+                
+                for (auto it_pair = all_conditions_maps->begin(); it_pair != all_conditions_maps->end(); ++it_pair )
```

Comment:
`auto const& it_pair`?
and maybe even `(auto const& it_pair : all_condition_maps)`

Reply:
Yes, that's the new standard, i can change it